packages/fiori/src/themes/UploadCollection.css

```diff
@@ -26,19 +17,15 @@
 	background-color: var(--sapGroup_ContentBackground);
 }
 
-.uc-no-files .icon-container {
-	height: 8rem;
-}
-
 .uc-no-files .icon-container [ui5-icon] {
-	font-size: 6rem;
-	width: 6rem;
-	height: 6rem;
+	font-size: 8rem;
```

Comment:
The ```font-size``` property is needed at all, because the size of the ```ui5-icon``` is set with width & height properties.